packages/trace-viewer/src/ui/errorsTab.tsx

```diff
@@ -55,9 +52,6 @@ export function useErrorsTabModel(model: modelUtil.MultiTraceModel | undefined):
 }
 
 function Error({ message, error, errorId, sdkLanguage, revealInSource }: { message: string, error: modelUtil.ErrorDescription, errorId: string, sdkLanguage: Language, revealInSource: (error: modelUtil.ErrorDescription) => void }) {
```

Comment:
Shall we rename this one to `ErrorView` or inline it back into `ErrorsTab` while we are here?

Reply:
it's hard to inline it because of the `useAsyncMemo`, but I renamed it.